samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -1441,3 +1441,8 @@ label#itemForm\:noFeedbackMsg {
 	list-style: none;
 	padding-left: 0;
 }
+
+.hidden {
+	display: none;
+	visibility: hidden;
+}
```

Comment:
Is there a bootstrap class you can use instead?

Reply:
`.hidden`, actually so we are going to remove this and update as fast as we can

Reply:
@ottenhoff done!